lte/gateway/c/session_manager/StatsPoller.cpp

```diff
@@ -13,17 +13,32 @@
 #include <stdint.h>
 #include <atomic>
 #include <memory>
+#include <vector>
 #include "StatsPoller.h"
-#define COOKIE 0
-#define COOKIE_MASK 0
+#define OVS_COOKIE_MATCH_ALL 0xffffffff
 
 namespace magma {
 
 void StatsPoller::start_loop(
     std::shared_ptr<magma::LocalEnforcer> local_enforcer,
     uint32_t loop_interval_seconds) {
+  // when no shard ids are available poll all UEs
+  // to poll by cookie pass in a vector of shard ids, where
+  // each element is a shard id that maps directly to a cookie
+  auto shard_tracker = local_enforcer->get_shard_tracker();
```

Comment:
Hmm I think it might be better to just pass shard_tracker in constructor to StatsPoller? adding a get method for it in LocalEnforcer seems convoluted